src/Steps/ReleaseManager.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Leviy\ReleaseTool\Steps;
```

Comment:
I don't think this class belongs in this namespace. Perhaps just put it in `Leviy\ReleaseTool`?